lib/dor/text_extraction/speech_to_text.rb

```diff
@@ -71,6 +71,11 @@ def output_location
         "#{job_id}/output"
       end
 
+      # look in resource structural metadata for a given filename and return it's cocina language (or nil if no language is set)
+      def language_tag(filename)
+        stt_files.find { |file| file.filename == filename }&.languageTag
```

Comment:
question ... do we want the default value to pass to whisper to be nil or `en` ?  this makes it nil, but we could make it `en`

Reply:
Preferably there would be no language parameter passed to the service if it is not known.

Reply:
Let me see if I can make that adjustment.

Reply:
changed - so ready for testing when the new container is ready